src/main/webapp/resources/js/pages/analysis/components/AnalysisShare.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Button, Checkbox, Card, Row, Alert } from "antd";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { AnalysisDetailsContext } from '../../../contexts/AnalysisDetailsContext';
+import { getI18N } from "../../../utilities/i18n-utilties";
+import { showNotification } from "../../../modules/notifications";
+
+import {
+  getSharedProjects,
+  updateSharedProjects,
+  saveToRelatedSamples
```

Comment:
`saveToRelatedSamples` is not used.

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a